sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -85,6 +87,17 @@ int main(int argc, char **argv) {
     init_uart();
   }
 
+  // The kJitterEnabled symbol defaults to false across all hardware platforms.
```

Comment:
@weicaiyang 
@timothytrippel 

should we actually move this to test_rom? Because technically that's where the jitter would be turned on right?
Long term when we want to measure total boot time, it will most accurate from there as well.

@alphan

Reply:
Yeah that probably makes more sense, since if we run a test with the mask ROM, it would be enabled once by the mask ROM and again by the OTTF. I submitted a PR with the fix here: #12459